css/globals.css

```diff
@@ -0,0 +1,61 @@
+:root {
+  --primary-color: #84a63c;
+  --secondary-color: #b7d93d;
+  --tertiary-color: #667303;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  font-family: "Courier New", Courier, monospace;
+  padding: 0;
+  margin: 0;
+}
+
+main {
+  margin: 100px;
```

Comment:
for mobile this is way to much space to the outside

Reply:
count and bookmark on your profile page are overlapping, they don't have enough room on iPhone SE

Reply:
same for about me

Reply:
How do I exclude that they overlap?